s/frontend/components/demo-auth/namaka-demo-auth.ts

```diff
@@ -60,6 +66,10 @@ export class NamakaDemoAuth extends mixinStandard<{
 						${isAdmin ?"(admin)" :null}
 						"${auth.user.profile.nickname}"
 					</p>
+					<button
```

Comment:
currently, this button only appears while the user is logged in.

but it should always be available, even when the user is logged out.